### PR TITLE
Add optional blackfire installation in docker

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | This PR adds blackfire when installing prestashop via docker, it is optional and disabled by default.
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Override the 3 blackfire environnement variable and check that blackfire is working (QA by dev, contacting me for getting a server id and token)
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/9388635380
| Fixed issue or discussion?     | #none
| Related PRs       | none
| Sponsor company   | PrestaShop

### CODE REVIEW

**Review by kpodemski — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]

**Comment by ps-jarvis** [comment]
QA approved, well done! Message to the maintainers: do not forget to milestone it before the merge.

**Review by jolelievre — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Pull request merged**
